corehq/apps/cloudcare/templates/form_entry/templates.html

```diff
@@ -222,7 +222,9 @@ <h3 class="caption" data-bind="html: header" tabindex="0"></h3>
       </div>
     </div>
     <div class="panel-footer">
-      <button class="btn btn-default btn-xs add" href="#" data-bind="click: newRepeat">{% trans "Add new repeat" %}</button>
+      <button class="btn btn-default btn-xs add" href="#"
+              data-bind="click: newRepeat, text: getTranslation('repeat.dialog.add.new', '{% trans 'Add new repeat' %}')"
```

Comment:
This won't parse if the HQ translation of "Add new repeat" contains a single quote. You can use `trans_html_attr` to get the right escaping.

Reply:
Thinking about this for 5 more seconds, will the second parameter ever get used? Translations already have default values, so it seems potentially confusing to have an additional default.

Reply:
I think there are different languages in the .po files vs the commcare-core translations? I wasn't sure how to check the overlap, and didn't want to break anything just in case.

Reply:
Mobile definitely has some kind of fallback, probably defaulting to English if the selected language doesn't exist. I'd test that out rather than add complexity to the code for an edge case that I don't think is real.

Reply:
hm, it doesn't seem to be on staging. Here's the sample app that I posted a screenshot of earlier https://staging.commcarehq.org/a/test-42/apps/view/4c06997fe58550220d11b15085247a46/form/15e9dbfe8af64d2abbe1609b9d5caab1/source/#form/outer_group/test

The default translation is being used for english, and the translations response has null (which means the translation doesn't exist I believe).

Reply:
Could it be a difference between `Localization.getWithDefault` that you're using in the new formplayer PR versus the `Localization.get` that commcare-android uses for that translation? Also, how are you "clearing" the translation?

Reply:
Ahh I didn't know about that, I'll try it next. The `self.parent.translation` is populated from the response json, so when you navigate to another page all elements there will have a different `self.parent.translation`.

Reply:
`Could it be a difference between Localization.getWithDefault that you're using in the new formplayer PR versus the Localization.get that commcare-android uses for that translation?`

Finally got a chance to look at this. The [getWithDefault](Could it be a difference between Localization.getWithDefault that you're using in the new formplayer PR versus the Localization.get that commcare-android uses for that translation?) that I'm using just is a wrapper around the `get` so that if it's not in there, the default value of null that I pass in is used instead of errorring. 

In any case, I think it's nice to pass in a default value in case there isn't one or nothing is returned, and stylistically in code we can more easily understand what the semantics of that text is. Also it'd be nice to not change the usability for languages with a .po translation there without understanding all the translation changes.